augur/augur_client.py

```diff
@@ -161,3 +190,6 @@ def _require_is_open(self):
   def _to_snake_case(self, text):
     s1 = re.sub('(.)([A-Z][a-z]+)', r'\1_\2', text)
     return re.sub('([a-z0-9])([A-Z])', r'\1_\2', s1).lower()
+
+  def _ensure_unix_timestamp(self, timestamp):
+    return datetime.fromtimestamp(int(str(timestamp)[:10]))
```

Comment:
What is the format used by Ethereum? Can you look into parsing it using ```datetime.strptime```?

Reply:
Ethereum should be storing timestamps in seconds, however, when I ran `load_market_info` through all markets I got some weird timestamps like:
```
1535141988837623
91268587776
91268587776
91268587776
91302002688
91302002688
91302002688
```

Reply:
It seems like these only appear on the Rinkeby test network. The first timestamp is NaN, the others are set to a far off date in the future. Running it on the Mainnet didn't produce any of these errors. Would it be safe to assume that the timestamp passed to `load_market_info` will be in the correct format?